WalletWasabi.Gui/Controls/WalletExplorer/HistoryTabViewModel.cs

```diff
@@ -105,7 +105,7 @@ private async Task TryRewriteTableAsync()
 				{
 					DateTime = txr.dateTime.ToLocalTime(),
 					Confirmed = txr.height.Type == HeightType.Chain,
-					Confirmations = txr.height.Type == HeightType.Chain ? Global.BitcoinStore.HashChain.TipHeight - txr.height.Value + 1 : 0,
+					Confirmations = txr.height.Type == HeightType.Chain ? (Global.BitcoinStore.HashChain.TipHeight - txr.height.Value) + 1 : 0,
```

Comment:
This doesn't fix anything:
![image](https://user-images.githubusercontent.com/127973/61834970-53ea6e00-ae50-11e9-8a14-2a82550e491f.png)

Reply:
Looks like I was getting a different error